src/app/authentication/authentication.component.ts

```diff
@@ -9,13 +10,24 @@ import { LoginAuthService } from "src/app/login-auth.service";
   styleUrls: ["./authentication.component.scss"]
 })
 export class AuthenticationComponent implements OnInit {
-  constructor(private loginAuthService: LoginAuthService) {
-    this.loginAuthService = loginAuthService;
-  }
+  constructor(
+    private loginAuthService: LoginAuthService,
+    public router: Router
+  ) {}
 
   ngOnInit() {}
 
   onSubmit(loginForm: any) {
-    this.loginAuthService.onSubmit(loginForm);
+    this.loginAuthService
+      .login(loginForm.value.username, loginForm.value.password)
+      .then(ok => {
+        if (ok === true) {
```

Comment:
you can right -> if (ok) 
no need to compare ok === true

Reply:
ok i will fix that.